ci-operator/step-registry/baremetalds/two-node/arbiter/workload-partitioning/baremetalds-two-node-arbiter-workload-partitioning-workflow.yaml

```diff
@@ -0,0 +1,31 @@
+workflow:
+  as: baremetalds-two-node-arbiter-workload-partitioning
+  steps:
+    allow_skip_on_success: true
+    allow_best_effort_post_steps: true
+    cluster_profile: equinix-edge-enablement
+    pre:
+      - chain: baremetalds-ipi-pre
+      - ref: openshift-cluster-bot-rbac
+      - ref: ipi-conf-cpu-partitioning-performance-profile
+    test:
+      - chain: baremetalds-ipi-test
+    post:
+      - chain: baremetalds-ipi-post
+    env:
+      PACKET_OS: rocky_9
+      CLUSTERTYPE: baremetal
+      RESERVED_CORES: "0-1"
+      ISOLATED_CORES: "2-3"
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4
+        NUM_MASTERS=2
+        MASTER_MEMORY=32768
+        NUM_ARBITERS=1
+        ARBITER_MEMORY=16384
+        NUM_WORKERS=0
+        ENABLE_ARBITER=true
```

Comment:
Lets remove this line, this is no longer needed, and lets add more CPU for the masters, add `MASTER_VCPU=8` and set `RESERVED_CORES: "0-3"` and `ISOLATED_CORES: "4-5"`